dotnet-testing/MockingWithNSubstitute/MockingWithNSubstitute.Tests/NotificationServiceTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using FluentAssertions;
+using NSubstitute;
+
+namespace MockingWithNSubstitute.Tests
+{
+    public class NotificationServiceTests
+    {
+        private readonly User _user;
+        private readonly IEmailService _emailService;
```

Comment:
This and the related tests should be in their own EmailServiceTests file, especially given that we have named this file "NotificationServiceTests"